server/text_generation_server/utils/paged_attention.py

```diff
@@ -1,13 +1,9 @@
 import torch
-from text_generation_server.utils.import_utils import (
-    IS_CUDA_SYSTEM,
-    IS_ROCM_SYSTEM,
-    IS_XPU_SYSTEM,
-)
+from text_generation_server.utils.import_utils import SYSTEM
 
 _PARTITION_SIZE = 512
 
-if IS_XPU_SYSTEM:
+if SYSTEM == "xpu":
```

Comment:
What's the motivation?

Reply:
Having a real enum, those are exclusive.
Meaning we cannot have both IS_XPU and IS_AMD at the same time (which was the case for the bools).

Reply:
String comparison is because python lacks good enum support unfortunately.

Reply:
very nitpick but `Enum System` couldn't work? `SYSTEM == System.CUDA`?

Reply:
Sure it can, enum are just painful to work with in Python. I just avoid them by default.
Had too many issues with them, where simple strings are just as fine (biggest caveat being the potential mistyping something, runtime is just as fast since strings are immutable)

Reply:
Have fun https://github.com/python/cpython/issues/93910  :)